cloud/ec2_util.go

```diff
@@ -326,6 +329,10 @@ func cacheHostData(ctx context.Context, h *host.Host, instance *ec2.Instance, cl
 		return errors.Wrap(err, "error updating host document in db")
 	}
 
+	if err := h.SetIPv4Address(*instance.PublicIpAddress); err != nil {
```

Comment:
Rather than calling a separate function for this, can we include it in CacheHostData? (the reason we don't do this for IPv6 is since we need to loop and decide if it needs to be set)

Reply:
made this function optional to only virtual workstations so I kept this function separate from the other cache host data

Reply:
Since we aren't making this only for virtual workstations now, we can just set `h.Ipv4Address = *instance.PublicIPAddress` above line 328 and then modify h.CacheHostData to also update ipv4 address.

Reply:
done